Makefile

```diff
@@ -236,13 +236,18 @@ profile:
 # generating makefiles to build the project.  For speed, it only does this when
 # ${build_dir}/Makefile doesn't already exist, in which case we rely on the
 # cmake generated makefiles to regenerate themselves when necessary.
+# When building missing dependencies, once configuration step completes,
+# rerun the configuration so that CMake rescans newly installed packages.
 config:
 	@ (if [ ! -e ${build_dir}/${BUILDSENTINEL} ] ; then \
 		${CMAKE} -E make_directory ${build_dir} ; \
 		cd ${build_dir} ; \
 		${CMAKE} -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE} \
 			 -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
 			 ${MY_CMAKE_FLAGS} ${working_dir} ; \
+		if [ "${BUILD_MISSING_DEPS}" = "1" ] ; then \
+				${CMAKE} ${working_dir} ; \
+			  fi ; \
```

Comment:
I'm fine with this change, at worst it's harmless, but I'm not sure I understand why a second invocation is needed. Can you ELI5?

Reply:
I may actually got to the bottom of the problem which makes the above change redundant.
The libutil target - (`setup_oiio_util_library` function) does not account for the include path to the "missing required" dependencies just built. At the build time, they are located under `./deps/dist/include`). The only include path currently set there is the final install prefix location.
So adding  `$<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/deps/dist/include>` seems to address it.